### PR TITLE
mimecast v2 - Fixed an issue where added users by domains showed an er…

### PR DESCRIPTION
**Pull request by bakatzir**
…ror though performed successfully.

## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/26154

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
@yuvalbenshalom confirmed solution with customer in the referenced issue.

File: Packs/Mimecast/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __MimecastV2__
+Fixed an issue where added users by emails showed an error though performed successfully.
```
Comment (amshamah419) [inline Packs/Mimecast/ReleaseNotes/1_1_0.md:4]: Fixed an issue where added users by emails returned an error although executed successfully.

Is “added users by emails” a command? If not try: “users added via emails”

Comment (bakatzir) [reply Packs/Mimecast/ReleaseNotes/1_1_0.md:4]: you can add a user via domain or via email. changing.

**Comment by bakatzir** [comment]
@yuvalbenshalom I added a UT, a change to the integration code was needed, please review.

**Pull request merged**